integration-tests/features/describe-stack-group.feature

```diff
@@ -1,25 +0,0 @@
-Feature: Describe stack_group
```

Comment:
the steps for these tests are in `integration-tests/steps/stacks.py`.  should we remove those as well?